src/main/java/request/HttpMethod.java

```diff
@@ -0,0 +1,22 @@
+package request;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum HttpMethod {
+
+    GET, POST;
+
+    private static final Map<String, HttpMethod> mappings = new HashMap<>();
+
+    static {
+        for (HttpMethod httpMethod : values()) {
+            mappings.put(httpMethod.name(), httpMethod);
+        }
+    }
```

Comment:
`of`를 위한 value cache 👍 
static block을 없애고 싶으면 stream을 이용할 수 있습니다.
```suggestion
    private static final Map<String, HttpMethod> mappings = Arrays.stream(values())
            .collect(Collectors.toMap(HttpMethod::name, Function.identity()));
```

Reply:
오 넵 감사합니다 identity 메소드에 대해 배울 수 있었습니다!😄